website/client/components/achievements/emptyArmory.vue

```diff
@@ -0,0 +1,48 @@
+<template lang="pug">
+  b-modal#empty-armory(:title="$t('modalAchievement')", size='md', :hide-footer="true", @hidden="reloadPage()")
+    .modal-body
+      .col-12
+        achievement-avatar.avatar
+      .col-6.offset-3.text-center
+        div(v-if='user.achievements.emptyArmoryLevel < 100')
+          | {{ $t('emptyArmoryAchievement', {number: user.achievements.emptyArmorys, level: user.achievements.emptyArmoryLevel}) }}
+        div(v-if='user.achievements.emptyArmoryLevel >= 100')
+          | {{ $t('emptyArmoryAchievement100', {number: user.achievements.emptyArmorys}) }}
```

Comment:
I get that this is modeled after the Orb of Rebirth, but given how little association there is between equipment and level, it seems really strange to track max level the same way. It might be more fun to track just how many pieces of equipment the user has dumped back into the void...